src/Core/Utilities/CoreHelpers.cs

```diff
@@ -459,18 +459,6 @@ public static string FormatLicenseSignatureValue(object val)
         return val.ToString();
     }
 
-    public static string GetVersion()
-    {
-        if (string.IsNullOrWhiteSpace(_version))
-        {
-            _version = Assembly.GetEntryAssembly()
```

Comment:
Should remove the unused field

Reply:
I believe this has already been removed 👍